`cargo install martin` fails on `actions-rust-lang/setup-rust-toolchain` GitHub actions runner

Hi!

The issue I describe below is, admittedly, a slightly specific use case, but the setup described has worked until v0.17.0 without a flaw (and I don’t see why it shouldn’t continue to work). 

I have a [GitHub action runner workflow](https://github.com/christophfink/radlkarte-at-tilejson/actions/runs/16766226856/workflow) set up to render an mbtiles database from a PostgreSQL dump, using martin. To achieve this, I first use the [actions-rust-lang/setup-rust-toolchain](https://github.com/actions-rust-lang/setup-rust-toolchain) to set up a rust environment on a vanilla ubuntu-latest runner, then run `cargo install martin`. This used to work until yesterday and fails now with an error related to `Etag`. 

Note this might be an error that occurs in my peculiar setup, only. Reporting it still, as it might point to an edge case that could re-surface elsewhere, as well.



```
error[E0423]: expected value, found struct `Etag`
Error:   --> /home/runner/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/martin-0.18.0/src/srv/fonts.rs:23:12
   |
23 |     wrap = "Etag",
   |            ^^^^^^ help: use struct literal syntax instead: `Etag { force_strong_etag: val }`
   |
  ::: /home/runner/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/actix-middleware-etag-0.4.6/src/lib.rs:55:1
   |
55 | pub struct Etag {
   | --------------- `Etag` defined here

error[E0423]: expected value, found struct `Etag`
Error:   --> /home/runner/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/martin-0.18.0/src/srv/tiles_info.rs:25:12
   |
25 |     wrap = "Etag",
   |            ^^^^^^ help: use struct literal syntax instead: `Etag { force_strong_etag: val }`
   |
  ::: /home/runner/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/actix-middleware-etag-0.4.6/src/lib.rs:55:1
   |
55 | pub struct Etag {
   | --------------- `Etag` defined here

error[E0423]: expected value, found struct `Etag`
Error:   --> /home/runner/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/martin-0.18.0/src/srv/sprites.rs:19:12
   |
19 |     wrap = "Etag"
   |            ^^^^^^ help: use struct literal syntax instead: `Etag { force_strong_etag: val }`
   |
  ::: /home/runner/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/actix-middleware-etag-0.4.6/src/lib.rs:55:1
   |
55 | pub struct Etag {
   | --------------- `Etag` defined here

error[E0423]: expected value, found struct `Etag`
Error:   --> /home/runner/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/martin-0.18.0/src/srv/sprites.rs:35:12
   |
35 |     wrap = "Etag"
   |            ^^^^^^ help: use struct literal syntax instead: `Etag { force_strong_etag: val }`
   |
  ::: /home/runner/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/actix-middleware-etag-0.4.6/src/lib.rs:55:1
   |
55 | pub struct Etag {
   | --------------- `Etag` defined here

error[E0423]: expected value, found struct `Etag`
Error:   --> /home/runner/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/martin-0.18.0/src/srv/sprites.rs:51:12
   |
51 |     wrap = "Etag",
   |            ^^^^^^ help: use struct literal syntax instead: `Etag { force_strong_etag: val }`
   |
  ::: /home/runner/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/actix-middleware-etag-0.4.6/src/lib.rs:55:1
   |
55 | pub struct Etag {
   | --------------- `Etag` defined here

error[E0423]: expected value, found struct `Etag`
Error:   --> /home/runner/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/martin-0.18.0/src/srv/sprites.rs:66:12
   |
66 |     wrap = "Etag",
   |            ^^^^^^ help: use struct literal syntax instead: `Etag { force_strong_etag: val }`
   |
  ::: /home/runner/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/actix-middleware-etag-0.4.6/src/lib.rs:55:1
   |
55 | pub struct Etag {
   | --------------- `Etag` defined here

error[E0423]: expected value, found struct `Etag`
Error:   --> /home/runner/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/martin-0.18.0/src/srv/styles.rs:19:12
   |
19 |     wrap = "Etag",
   |            ^^^^^^ help: use struct literal syntax instead: `Etag { force_strong_etag: val }`
   |
  ::: /home/runner/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/actix-middleware-etag-0.4.6/src/lib.rs:55:1
   |
55 | pub struct Etag {
   | --------------- `Etag` defined here

For more information about this error, try `rustc --explain E0423`.
error: could not compile `martin` (lib) due to 7 previous errors
warning: build failed, waiting for other jobs to finish...
error: failed to compile `martin v0.18.0`, intermediate artifacts can be found at `/tmp/cargo-installeMzgQP`.
To reuse those artifacts with a future compilation, set the environment variable `CARGO_TARGET_DIR` to that path.
Error: Process completed with exit code 101.
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -6,7 +6,7 @@ members = ["martin", "martin-tile-utils", "mbtiles"]
 edition = "2024"
 license = "MIT OR Apache-2.0"
 repository = "https://github.com/maplibre/martin"
-rust-version = "1.85"
+rust-version = "1.86" # aws-sdk
 homepage = "https://martin.maplibre.org/"
 
 [workspace.lints.rust]

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -6,7 +6,7 @@ members = ["martin", "martin-tile-utils", "mbtiles"]
 edition = "2024"
 license = "MIT OR Apache-2.0"
 repository = "https://github.com/maplibre/martin"
-rust-version = "1.86" # aws-sdk
+rust-version = "1.86"
 homepage = "https://martin.maplibre.org/"
 
 [workspace.lints.rust]

--- a/martin/Cargo.toml
+++ b/martin/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "martin"
-version = "0.18.0"
+version = "0.18.1"
 authors = [
   "Stepan Kuzmin <to.stepan.kuzmin@gmail.com>",
   "Yuri Astrakhan <YuriAstrakhan@gmail.com>",

--- a/martin/martin-ui/package.json
+++ b/martin/martin-ui/package.json
@@ -61,5 +61,5 @@
     "test:watch": "jest --watch",
     "type-check": "tsc --noEmit"
   },
-  "version": "0.18.0"
+  "version": "0.18.1"
 }

